crates/cli/README.md

```diff
@@ -194,4 +194,4 @@ If you don't need PID1 support, or if you're doing something that conflicts with
 
 [Eyra](https://github.com/sunfishcode/eyra) is a system to build Linux programs with no dependency on C code (in the libc path). To build Watchexec like this, use `--features eyra` and a Nightly compiler.
 
-This feature also lets you get early logging into program startup, with `RUST_LOG=trace`.
+This feature also lets you get early logging into program startup, with `WATCHEXEC_LOG=trace`.
```

Comment:
eyra specifically [has its own logger](https://github.com/sunfishcode/eyra#optional-logging) that responds to `RUST_LOG` (and runs before `main()`, so there's no runtime customisation possible here).

Reply:
Good catch!